packages/api-plugin-sequences/src/simpleSchemas.js

```diff
@@ -0,0 +1,7 @@
+import SimpleSchema from "simpl-schema";
+
+export const Sequence = new SimpleSchema({
+  shopId: String,
+  entity: String,
+  value: SimpleSchema.Integer
```

Comment:
Consider adding `min: 0` validation

Reply:
@aldeed Could not think of what to write tests for since it relies to heavily on the db which would need to mocked